FeedStoreChallenge/ManagedCache.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  ManagedCache.swift
+//  FeedStoreChallenge
+//
+//  Created by Prabhat Tiwari on 20/08/21.
+//  Copyright © 2021 Essential Developer. All rights reserved.
+//
+
+import CoreData
+
+@objc(ManagedCache)
+internal class ManagedCache: NSManagedObject {
```

Comment:
The convention in this codebase is to omit the default access level keyword `internal`. Also, `ManagedCache` is not supposed to be subclassed, so it should be marked as `final`:
```suggestion
final class ManagedCache: NSManagedObject {
```